hack/extra_rpm_repositories.yml

```diff
@@ -1,67 +1,8 @@
 # Update this list for any additional repos that should be added to the list of known repositories.
-extras:
-  -  "ubi-8-baseos-rpms"
-  -  "ubi-8-baseos-debug-rpms"
-  -  "ubi-8-baseos-source"
-  -  "ubi-8-appstream-rpms"
-  -  "ubi-8-appstream-debug-rpms"
-  -  "ubi-8-appstream-source"
-  -  "ubi-8-codeready-builder-rpms"
-  -  "ubi-8-codeready-builder"
-  -  "ubi-8-codeready-builder-debug-rpms"
-  -  "ubi-8-codeready-builder-source"
-  -  "ubi-9-baseos-rpms"
-  -  "ubi-9-baseos-debug-rpms"
-  -  "ubi-9-baseos-source"
-  -  "ubi-9-baseos-source-rpms"
-  -  "ubi-9-appstream-rpms"
-  -  "ubi-9-appstream-debug-rpms"
-  -  "ubi-9-appstream-source"
-  -  "ubi-9-appstream-source-rpms"
-  -  "ubi-9-codeready-builder-rpms"
-  -  "ubi-9-codeready-builder"
-  -  "ubi-9-codeready-builder-debug-rpms"
-  -  "ubi-9-codeready-builder-source"
-  -  "ubi-9-codeready-builder-source-rpms"
-  -  "ubi-10-baseos-rpms"
-  -  "ubi-10-baseos-debug-rpms"
-  -  "ubi-10-baseos-source"
-  -  "ubi-10-baseos-source-rpms"
-  -  "ubi-10-appstream-rpms"
-  -  "ubi-10-appstream-debug-rpms"
-  -  "ubi-10-appstream-source"
-  -  "ubi-10-appstream-source-rpms"
-  -  "ubi-10-codeready-builder-rpms"
-  -  "ubi-10-codeready-builder"
-  -  "ubi-10-codeready-builder-debug-rpms"
-  -  "ubi-10-codeready-builder-source"
-  -  "ubi-10-codeready-builder-source-rpms"
-  -  "rhel-8-appstream-rpms"
-  -  "rhel-8-appstream-source-rpms"
-  -  "rhel-8-baseos-rpms"
-  -  "rhel-8-baseos-source-rpms"
-  -  "rhel-8-codeready-builder-rpms"
-  -  "rhel-8-codeready-builder-source-rpms"
-  -  "rhel-9-baseos-rpms"
-  -  "rhel-9-baseos-source-rpms"
-  -  "rhel-9-appstream-rpms"
-  -  "rhel-9-appstream-source-rpms"
-  -  "rhel-9-codeready-builder-rpms"
-  -  "rhel-9-codeready-builder-source-rpms"
-  -  "rhel-9-rt-rpms"
-  -  "rhel-9-rt-source-rpms"
-  -  "rhel-10-baseos-rpms"
-  -  "rhel-10-baseos-source-rpms"
-  -  "rhel-10-appstream-rpms"
-  -  "rhel-10-appstream-source-rpms"
-  -  "rhel-10-codeready-builder-rpms"
-  -  "rhel-10-codeready-builder-source-rpms"
-  -  "rhel-10-rt-rpms"
-  -  "rhel-10-rt-source-rpms"
-  -  "rhel-10-flatpak-app-build"
-  -  "baseos"
-  -  "baseos-sources"
-  -  "appstream"
-  -  "appstream-sources"
-  -  "codeready-builder"
-  -  "codeready-builder-sources"
+#
+# As of KONFLUX-6218, this should not be used. At least, repositories should only be added here with extreme caution.
+#
+# An image that contains an rpm that is associated with a valid repository id in the public mapping will not be
+# properly understood by clair and third-party scanners.
+
+extras: []
```

Comment:
Do we want to keep the `extras` functionality as an escape valve just in case :tm: ?

If we're ready to drop this altogether, let's simplify [update-known-rpm-repositories.sh](https://github.com/release-engineering/rhtap-ec-policy/blob/main/hack/update-known-rpm-repositories.sh).

Reply:
My gut says "keep it as an escape".